components/server/src/prebuilds/github-enterprise-app.ts

```diff
@@ -258,7 +260,9 @@ export class GitHubEnterpriseApp {
 
     private async findProjectOwners(cloneURL: string): Promise<{ users: User[]; project: Project } | undefined> {
         try {
-            const projects = await this.projectService.findProjectsByCloneUrl(SYSTEM_USER, cloneURL);
+            const projects = await runWithSubjectId(SubjectId.fromUserId(SYSTEM_USER), async () =>
+                this.projectService.findProjectsByCloneUrl(SYSTEM_USER, cloneURL),
```

Comment:
How can we know that we need to do such thing? i.e. Verify if we have anywhere not add it.

Find where we use with SYSTEM_USER requestId userId?

Reply:
> How can we know that we need to do such thing? i.e. Verify if we have anywhere not add it.

Excellent question! I have a follow-up PR that checks this in `Authorizer`, increments a metric and logs an error. That's what we need to drive to 0 before we can use this for rollout authorization. :100: